onvm/onvm_nflib/onvm_nflib.c

```diff
@@ -609,19 +613,27 @@ onvm_nflib_get_nf(uint16_t id) {
         return &nfs[id];
 }
 
+void
+onvm_nflib_stop(struct onvm_nf_local_ctx *nf_local_ctx) {
+        /* Terminate children */
+        onvm_nflib_terminate_children(nf_local_ctx->nf);
+        /* Stop and free */
+        onvm_nflib_cleanup(nf_local_ctx);
+}
+
 struct onvm_configuration *
 onvm_nflib_get_onvm_config(void) {
         return onvm_config;
 }
 
 void
-onvm_nflib_set_setup_function(struct onvm_nf_info *info, setup_func setup) {
-        nfs[info->instance_id].nf_setup_function = setup;
+onvm_nflib_set_setup_function(struct onvm_nf *nf, setup_func setup) {
```

Comment:
the type `setup_func` is ambiguous when used in places like this.  can you please rename the typedef to something more clear?

(the name of the field, `nf->functions.setup` is fine because the `onvm_nf` struct's fields make it descriptive)

Reply:
Does nf_setup_func sound better? Or maybe just changing the API to `onvm_nflib_nf_set_setup_function`